indicators/management/commands/create_participant_count_indicators.py

```diff
@@ -1,230 +1,194 @@
-                    indicator.delete()
-                    deleted_count += 1
-                except Indicator.DoesNotExist:
-                    pass
-            print(f'Option1 indicators deleted for {country.country}: {deleted_count}')
+            disagg_sector_without_double, created = DisaggregationType.objects.get_or_create(
+                disaggregation_type='Sectors Direct without double counting',
+                country=None,
+                global_type=DisaggregationType.DISAG_PARTICIPANT_COUNT,
+                defaults={
+                    'is_archived': False,
+                    'selected_by_default': True,
+                }
+            )
+            if created:
+                for i, label in enumerate(sectors_list):
+                    DisaggregationLabel.objects.create(
+                        label=label, disaggregation_type=disagg_sector_without_double, customsort=i + 1)
+                print('Created disaggregation type: Sectors Direct without double counting')
+            else:
+                print('This disaggregation type already existed: Sectors Direct without double counting')
+
+            actual_disagg_labels = ['Direct', 'Indirect']
+            disagg_actual_with_double, created = DisaggregationType.objects.get_or_create(
+                disaggregation_type='Actual with double counting',
+                country=None,
+                global_type=DisaggregationType.DISAG_PARTICIPANT_COUNT,
+                defaults={
+                    'is_archived': False,
+                    'selected_by_default': True,
+                }
+            )
+            if created:
+                for i, label in enumerate(actual_disagg_labels):
+                    DisaggregationLabel.objects.create(
+                        label=label, disaggregation_type=disagg_actual_with_double, customsort=i + 1)
+                print('Created disaggregation type: Actual with double counting')
+            else:
+                print('This disaggregation type already existed: Actual with double counting')
 
+            disagg_actual_without_double, created = DisaggregationType.objects.get_or_create(
+                disaggregation_type='Actual without double counting',
+                country=None,
+                global_type=DisaggregationType.DISAG_PARTICIPANT_COUNT,
+                defaults={
+                    'is_archived': False,
+                    'selected_by_default': True,
+                }
+            )
+            if created:
+                for i, label in enumerate(actual_disagg_labels):
+                    DisaggregationLabel.objects.create(
+                        label=label, disaggregation_type=disagg_actual_without_double, customsort=i + 1)
+                print('Created disaggregation type: Actual without double counting')
+            else:
+                print('This disaggregation type already existed: Actual without double counting')
```

Comment:
I see actuals with and without double counting but do we need to also specify indirect and direct here?